qemu/tests/cfg/blockdev_snapshot.cfg

```diff
@@ -41,3 +41,12 @@
         enable_ceph_sn1 = no
         enable_ceph_data = yes
         image_format_data = raw
+    nbd:
+        enable_nbd_sn1 = no
+        enable_nbd_data = yes
+        image_format_data = raw
+        nbd_port_data = 10810
```

Comment:
Discussed in the nbd setup doc, we may also need to update the ports used in all test cases

Reply:
Have updated all of them, thanks zhenchao.